CHANGLOG.md

```diff
@@ -1,3 +1,19 @@
+## v2.1.0 / 2022-11-29
+
+### FEATURES
+
+- Add support to send notifications to AWS SMS (#159). @Bennu-Li
+- Use the `TZ` environment variable instead of the host path `/etc/localtime` to set the time zone (#148). @ctrought
+
+### Enhancement
+
+- Support to route notifications to specified tenants (#163). @wanjunlei
```

Comment:
Looks like features and enhancements can be adjusted like below:
```yaml
### FEATURES
- Support sending notifications to AWS SMS (#159). @Bennu-Li
- Support routing notifications to specified tenants (#163). @wanjunlei

### Enhancements
- Support using the `TZ` environment variable instead of the host path `/etc/localtime` to set the time zone (#148). @ctrought
- Enhanced notification template supports automatic message selection based on notification language (#160). @wanjunlei

### BUGFIX

- Fixed `Feishu` does not support a short template name (#152). @wanjunlei
```